src/events/components/Attendance/UnattendButton.tsx

```diff
@@ -11,7 +12,11 @@ interface IAttendButtonProps {
 
 const UnattendButton: FC<IAttendButtonProps> = ({ eventId, isOnWaitList, waitListNumber }) => {
   const dispatch = useDispatch();
-  const signOff = () => dispatch(removeAttendeeByEventId(eventId));
+  const [addMessage] = useToast({ type: 'success', duration: 5000 });
+  const signOff = async () => {
+    await dispatch(removeAttendeeByEventId(eventId));
+    addMessage('Du har blitt meldt av arrangementet');
```

Comment:
Can we somehow check if the attendee _actually_ was removed? In case things fail?